enterprise_access/apps/api_client/lms_client.py

```diff
@@ -17,6 +17,73 @@ class LmsApiClient(BaseOAuthClient):
     """
     enterprise_api_base_url = settings.LMS_URL + '/enterprise/api/v1/'
     enterprise_learner_endpoint = enterprise_api_base_url + 'enterprise-learner/'
+    enterprise_customer_endpoint = enterprise_api_base_url + 'enterprise-customer/'
+
+    def get_enterprise_customer_data(self, enterprise_customer_uuid):
+        """
+        Gets the data for an EnterpriseCustomer for the given uuid.
+
+        Arguments:
+            enterprise_customer_uuid (string): id of the enterprise customer
+        Returns:
+            dictionary containing enterprise customer metadata
+        """
+
+        try:
+            endpoint = f'{self.enterprise_customer_endpoint}{enterprise_customer_uuid}'
+            response = self.client.get(endpoint, timeout=settings.LMS_CLIENT_TIMEOUT)
+        except requests.exceptions.HTTPError as exc:
+            logger.exception(exc)
+            raise
+
+        return response.json()
```

Comment:
Why not move the json parsing up and log that as well? Also we know that the method can raise at least one exception, this should be covered in the docs.

Reply:
Hmm, do you mean like adding in a `logger.info(response.json())` type line to the except block? or having one outside of the accept block? both?

In the case of "log it each time" and not just the except block, I think eventually there would be cost implications, and also just really noisy. If you just meant the except block, I think am okay with adding that there.

And in any case, I can update the doc string